packages/driver/cypress/integration/e2e/multidomain_spec.ts

```diff
@@ -1,22 +1,23 @@
 // FIXME: Skip this for now since it's flaky
 it.skip('verifies initial implementation of sibling iframe and switchToDomain', (done) => {
   top.addEventListener('message', (event) => {
-    if (event.data && event.data.text) {
-      expect(event.data.text).to.equal('Foo')
+    if (event.data && event.data.textFromParagraph !== undefined) {
       expect(event.data.host).to.equal('127.0.0.1:3501')
+      expect(event.data.textFromParagraph).to.equal('From a secondary domain with window.Cypress')
       done()
     }
   }, false)
 
-  cy.state('anticipateMultidomain', true)
   cy.viewport(900, 300)
   cy.visit('/fixtures/multidomain.html')
+  // @ts-ignore
+  cy.anticipateMultidomain()
```

Comment:
This is a temporary measure to enable setting up the cross-domain bridge iframe. Once we tackle #17252, we'll look ahead to the `cy.switchToDomain` command and this will no longer be required.